src/selectors/activities.js

```diff
@@ -0,0 +1,9 @@
+export default (activities, {text}) => {
+  return activities.filter((activity) => {
+    const textMatchName = activity.name.toLowerCase().includes(text.toLowerCase());
```

Comment:
try to create a function, which gets name/day/teacher/ etc and text and returns what it should returns. 
Target of this selector is not clear for me. Can you elaborate?

Reply:
So i destructurized activity, hope this helps a bit. This selector is here to filter out activities with passed into input text. Not sure if i can see a way to simplify this... Also i don't think it's that over-complicated. What do u think?

Reply:
put to some todos, we will come back to this